session/session.go

```diff
@@ -0,0 +1,83 @@
+// This package is for all 'session' related code.
+package session
+
+import (
+	"fmt"
+	"github.com/pocket_network/pocket-core/node"
+)
+
+var (
+	globalSessionList *sessionList
+)
+/*
+This is the session structure.
+ */
+ // TODO consider converting validators and servicers to map
```

Comment:
I think this should actually be done as a map, to avoid possible duplications, or a Set, look here for reference: https://softwareengineering.stackexchange.com/questions/177428/sets-data-structure-in-golang